src/pages/archive/Archive.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { Aside, Note } from "../../components";
+import { useArchiveContext, useAuthContext } from "../../context";
+export function Archive() {
+  const { archives, setArchives, unArchiveHandler} = useArchiveContext();
+  const { token } = useAuthContext();
+  const [editNoteBtn, setEditNoteBtn] = useState(false);
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/archives", {
+          headers: {
+            authorization: token,
+          },
+        });
+        if (response.status === 200) {
+          setArchives(response.data.archives);
+        }
+      } catch (err) {
+        console.error(err);
```

Comment:
I think it would be more consistent to move this function in `archive-context.js` as rest of the API calls related to archive are also in that file.